jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -623,6 +623,7 @@ Compatibility notes:
 ## OS-specific notes
 - Support for Apple silicon/M1.
 - Support for 32-bit RISC-V, refs [#16231](https://github.com/nim-lang/Nim/pull/16231).
```

Comment:
how are M1, 32-bit RISC-V and armv8l OS specific?

Reply:
changed title to: `## OS/platform-specific notes`

@Yardanico feel free to suggest alternatives